src/dynamicTypes/index.js

```diff
@@ -0,0 +1,36 @@
+const {
+  getPackageRawDeclaration,
+  getPackagesTypeDeclarations
+} = require("./npmDependencies");
+const {
+  getPageElementsRawDeclarations,
+  getPageElementsTypeDeclarations
+} = require("./elementsMap");
+const {
+  getWidgetRawDeclaration,
+  getWidgetsTypeDeclarations
+} = require("./widget");
+const { getJswRawDeclaration, getJswTypeDeclarations } = require("./jswFiles");
+const getJsonTypeDeclarations = require("./jsonFiles");
+
+module.exports = {
+  packages: {
+    getRaw: getPackageRawDeclaration,
```

Comment:
you can go one step further and make each file export getRaw & getFiles, so you don't need to split them here
```
const npmDependencies = require("./npmDependencies")
...
module.exports = {npmDependencies, ...}
```

Reply:
Nice, done